config.proto

```diff
@@ -17,18 +17,34 @@ message AgentConfig {
     // reporting holds the reporting settings for the agent
     Reporting reporting = 2;
 
+    // opa describes the setting related to the Open Policy Agent
+    Opa opa = 4;
```

Comment:
rather than have this in the top level, I'm thinking it makes more sense to move it into the Reporting message

Reply:
👍  moved there